src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/editor/labeler/marker/Marker.kt

```diff
@@ -597,6 +606,7 @@ private fun MarkerState.handleCursorPress(
     event: PointerEvent,
     labelerConf: LabelerConf,
     appConf: AppConf,
+    editorState: EditorState,
 ) {
     val action = keyboardState.getEnabledMouseClickAction(event) ?: return
     if (action.canMoveParameter()) {
```

Comment:
I think we need to add another `mouse action` instead of handling it over the existing moving actions, right now, if we enable this, it's very difficult to drag parameter lines, because when you press, it immediately causes the jumping.

Also, it might be better UX if we handle it on mouse release (up), not mouse press (down).

Even if we use another `mouse action`, we should note that the action def might be the same for multiple actions (e.g. we can simply set `Left Click` on this new action).
So, e.g. when we handle it on mouse release, we need to check whether this release event is a finishing event of a dragging, and only handle when it's not.